changelogs/fragments/100-more-other-projects.yml

```diff
@@ -0,0 +1,5 @@
+minor_changes:
+  - Allow to extract other project versions for Python projects from PEP 621 conformant ``pyproject.toml``
+    (https://github.com/ansible-community/antsibull-changelog/pull/100).
+  - Allow to extract other project versions for JavaScript / TypeScript projects form ``package.json``
```

Comment:
```suggestion
  - Allow to extract other project versions for JavaScript / TypeScript projects from ``package.json``
```

Reply:
Fixed in e1a6c248a16f7408f0bb1fc4ad69c16889621b6e.